src/Dashboard/EpochProgress.tsx

```diff
@@ -35,7 +35,7 @@ export const EpochProgress = ({percentage, currentEpoch, endTime}: Props) => {
           <View style={styles.row}>
             <Text style={styles.label}>{intl.formatMessage(messages.endsInLabel)}:</Text>
             <View style={styles.timeWrapper}>
-              {endTime.d && <Text style={styles.timeBlock}>{endTime.d}</Text>}
+              {endTime.d != null && <Text style={styles.timeBlock}>{endTime.d}</Text>}
```

Comment:
what does this change?

Reply:
Text strings must be rendered within a component.